### DISCUSSION TITLE
Custom extension content (e.g. floating-menu)

### DISCUSSION THREAD

**Discussion by jloganolson** [ueberdosis/tiptap#1798] [General Questions] [answered]
In the floating menu docs, the first example uses a component-like syntax to supply the element for tippy:
```
<floating-menu :editor="editor" v-if="editor">
 ...
</floating-menu>
```
How is that content making it's way to this.options.element without adding it to the configuration? I can't figure it out!

**Comment by hanspagel** [comment, marked as answer]
I think it’s this line: https://github.com/ueberdosis/tiptap/blob/main/packages/vue-3/src/FloatingMenu.ts#L52

**Reply by jloganolson** [reply]
That was totally it - thanks for the quick response!

FWIW, it would have been very helpful if the git links in the floating menu section of the tiptap.dev docs pointed to this. I had no idea about this vue-3 folder (I have only been looking in the commands extension folder the past few days).
